execution/executor-service/src/remote_state_view.rs

```diff
@@ -142,6 +187,27 @@ impl RemoteStateViewClient {
         let request_message = bcs::to_bytes(&request).unwrap();
         sender.send(Message::new(request_message)).unwrap();
     }
+
+    pub fn print_info(&self) {
+        info!(
+            "REMOTE_EXECUTOR: Total approx get prefetched val time is {} s",
```

Comment:
nit: I think it's desired to do 
```Rust
info!(
    prefetch_value_time = TIMER1.get_sample_sum(),
    kv_response_time = TIME2.get_sample_sum(),
    ...
    "whatever event happened.",
)
```

Reply:
We don't really need these log lines.. let's remove them and build a dashboards for these counters instead :)